app/scripts/controllers/settings.js

```diff
@@ -87,6 +87,21 @@ angular.module('openshiftConsole')
           Logger.log("quotas", $scope.quotas);
         });
 
+        DataService.list("appliedclusterresourcequotas", context, function(quotas) {
+          $scope.clusterQuotas = quotas.by("metadata.name");
+          $scope.emptyMessageQuotas = "There are no resource quotas set on this project.";
```

Comment:
`DataService.list` doesn't return a promise, does it? Then we could use `$q.all` to set the message.